extensions/podman/src/registry-setup.ts

```diff
@@ -196,4 +200,35 @@ export class RegistrySetup {
       });
     });
   }
+
+  protected checkAuthOptions(authOptions: extensionApi.RegistryCreateOptions): Promise<void | Error> {
+    return execPromise(getPodmanCli(), [
```

Comment:
would be nice to use REST API
https://docs.docker.com/registry/spec/auth/token/

Reply:
updated